templates/advantage/table/_manage-monthly.html

```diff
@@ -0,0 +1,72 @@
+<div class="row col-start-large-2 col-10" id="view-mode--{{contract["contractInfo"]["id"]}}{% if mobile %}--mobile{% endif %}">
```

Comment:
Row and cols perform different tasks. Can you nest them instead of using them together please.

Reply:
In stead of parsing this out when processing this contract Id. It would be clearer if you add a data attr or something like: 
```
data-viewport="small"
```

Mobile is not always correct as it could be a narrow window on a screen.